draft-ietf-rtcweb-jsep.xml

```diff
@@ -3419,7 +3421,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         false.</t>
 
         <t>The following steps are performed at the media engine level
-        to apply a remote description.</t>
+        to apply a remote description.  If an error is returned, the
+        session MUST be restored to the state it was in before
```

Comment:
the text here is ambiguous regarding whether canTrickle is reverted - need to resolve this.